src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -201,6 +201,15 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
                 {
                     this.Init(localTransport, _launchOptions);
                 }
+
+                // Only need to know the debugger pid on Linux and OS X local launch to detect whether
+                // the debugger is closed. If the debugger is not running anymore, the response (^exit)
+                // to the -gdb-exit command is faked to allow MIEngine to shut down.
+                _localDebuggerPid = localTransport.DebuggerPid;
+
+                // We are launching the debugger in the background, so the console will have
+                // "[job number] pid". So we clear that.
+                ResetConsole();
```

Comment:
resetting the terminal is very expensive. We had to do it that way to get rid of gdb's bogus error on launch. However, I think it might be overkill in this case. Let's try clear instead.